content/hi/docs/contribute/style/page-content-types.md

```diff
@@ -0,0 +1,221 @@
+
+| Page section  |
+|---------------|
+| overview      |
+| prerequisites |
+| steps         |
+| discussion    |
+| whatsnext     |
+
+कार्य पृष्ठ में `overview`, `steps` और `discussion` अनुभाग टिप्पणियों के रूप में दिखाई देते हैं।
+आप अपने पेज में `prerequisites` और `whatsnext` अनुभाग जोड़ सकते हैं `heading`
+शोर्ट के साथ।
+
+प्रत्येक अनुभाग में, अपनी सामग्री लिखें। निम्नलिखित दिशानिर्देशों का प्रयोग करें:
+
+- कम से कम H2 शीर्षकों का उपयोग करें (दो प्रमुख `#` वर्णों के साथ)। खंड
+  स्वयं को टेम्पलेट द्वारा स्वचालित रूप से शीर्षक दिया जाता है।
+- `overview` के लिए, पूरे विषय के लिए संदर्भ सेट करने के लिए एक अनुच्छेद का उपयोग करें।
+- `prerequisites` के लिए, जब संभव हो बुलेट सूचियों का उपयोग करें। अतिरिक्त जोड़ना शुरू करें
+  `include` के नीचे पूर्वापेक्षाएँ। डिफ़ॉल्ट पूर्वापेक्षाएँ एक चल रहे क्लस्टर कुबेरनेट्स शामिल हैं।
+- `steps` के लिए, क्रमांकित सूचियों का उपयोग करें।
+- चर्चा के लिए, कवर की गई जानकारी का विस्तार करने के लिए सामान्य सामग्री का उपयोग करें
+  `steps` में।
+- `Whatsnext` के लिए, अधिकतम पाँच विषयों की एक बुलेट सूची दें, जो पाठक हो सकते हैं
+  आगे पढ़ने में दिलचस्पी है।
+
+प्रकाशित कार्य विषय का एक उदाहरण है [Using an HTTP proxy to access the Kubernetes API](/docs/tasks/extend-kubernetes/http-proxy-access-api/).
+
+### ट्यूटोरियल
+
+एक ट्यूटोरियल पेज दिखाता है कि किसी एकल से बड़े लक्ष्य को कैसे पूरा किया जाए
+काम। आमतौर पर एक ट्यूटोरियल पेज में कई सेक्शन होते हैं, जिनमें से प्रत्येक में एक होता है
+चरणों का क्रम। उदाहरण के लिए, एक ट्यूटोरियल a . का पूर्वाभ्यास प्रदान कर सकता है
+कोड नमूना जो कुबेरनेट्स की एक निश्चित विशेषता को दर्शाता है। ट्यूटोरियल कर सकते हैं
+सतह-स्तरीय स्पष्टीकरण शामिल करें, लेकिन संबंधित अवधारणा विषयों से लिंक होना चाहिए
+गहरी व्याख्या के लिए।
+
+एक नया ट्यूटोरियल पेज लिखने के लिए, की एक उपनिर्देशिका में एक मार्कडाउन फ़ाइल बनाएं
+`/content/en/docs/tutorials` directory, निम्नलिखित विशेषताओं के साथ:
+
+| Page section  |
+|---------------|
+| overview      |
+| prerequisites |
+| objectives    |
+| lessoncontent |
+| cleanup       |
+| whatsnext     |
+
+ट्यूटोरियल पेज में 'overview', 'objectives' और 'lessoncontent' अनुभाग टिप्पणियों के रूप में दिखाई देते हैं।
+आप अपने पेज में `prerequisites`, `cleanup` और `whatsnext` अनुभाग जोड़ सकते हैं
+`heading` शोर्ट के साथ।
+
+प्रत्येक अनुभाग में, अपनी सामग्री लिखें। निम्नलिखित दिशानिर्देशों का प्रयोग करें:
+
+- कम से कम H2 शीर्षकों का उपयोग करें (दो प्रमुख `#` वर्णों के साथ)। खंड
+  स्वयं को टेम्पलेट द्वारा स्वचालित रूप से शीर्षक दिया जाता है।
+- `overview` के लिए, पूरे विषय के लिए संदर्भ सेट करने के लिए एक अनुच्छेद का उपयोग करें।
+- `prerequisites` के लिए, जब संभव हो बुलेट सूचियों का उपयोग करें। अतिरिक्त जोड़ें
+  पूर्वापेक्षाएँ डिफ़ॉल्ट रूप से शामिल लोगों के नीचे।
+- `objectives` के लिए, बुलेट सूचियों का उपयोग करें।
+- `lessoncontent` के लिए, क्रमांकित सूचियों और वर्णनात्मक सामग्री के मिश्रण का उपयोग इस प्रकार करें
+  उचित।
+- `cleanup` के लिए, क्रमांकित सूचियों का उपयोग करके सफाई के चरणों का वर्णन करें
+  कार्य पूरा करने के बाद क्लस्टर की स्थिति।
```

Comment:
- क्रमांकित सूचियों और कथा सामग्री के मिश्रण का उपयुक्त रूप में उपयोग करें।  might be better